PyDelhiConf Material/slides/intro.tex

```diff
@@ -60,7 +60,7 @@
       ``SymPy is an open source Python library for symbolic mathematics. It aims to
       become a full-featured computer algebra system (CAS) while keeping the code as
       simple as possible in order to be comprehensible and easily extensible. SymPy
-      is written entirely in Python and does not require any external libraries.''
+      is written entirely in Python and does not require any non-Python libraries.''
```

Comment:
Technically it's right, though the current release, 0.7.6.1, does not require any dependency as mpmath is included. Though we have to explicitly mention it for 1.0, so it's a good idea to mention.